### PR TITLE
add csharp-colors project

### PR DESCRIPTION
**Pull request by Logerfo**

### CODE REVIEW

**Comment by deammer** [comment]
Hi @Logerfo! Thank you for your contribution!

I noticed that the Netlify build failed for this PR because the CodeSee map URL didn't match the expected format. I fixed the issue in #43. If you rebase on top of the latest `main` branch and push your code again, that should clear up the problem.

Let me know if you run into any issues!

**Comment by Logerfo** [comment]
@deammer Hello! Thank you for rebasing. Also, let me introduce myself, as I should've done earlier: I authored a few VSCode extensions and Github Actions and have been invited to apply projects to oss-port. I chose csharp-colors to begin with, but I might add other projects as well. My only concern at this moment is that I'm currently not exactly able to make time to work on open source development.

File: projects/Logerfo/csharp-colors.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+description: Visual Studio Code extension that adds colored border around C# colors in the text editor.
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - TypeScript
+tags: # A list of keywords relevant to the contributors for this project
+  - c#
+  - csharp
+  - color
+  - vscode
+  - extension
```
Comment (deammer) [inline projects/Logerfo/csharp-colors.mdx:13]: **suggestion:** for the sake of discoverability, I would suggest modifying the tags for your project:

```suggestion
tags: # A list of keywords relevant to the contributors for this project
  - C#
  - VS Code extension
```

**Review by deammer — APPROVED** [review]
@Logerfo thank you for your contribution! This is the first C# project on OSS Port, which is very exciting. Your project will be live in a few minutes :)

**Comment by Logerfo** [comment]
@deammer Thanks! Well, it's actually not a C# project. It's made for C# devs, but it's written in TypeScript :)

**Pull request merged**